GAE/src/com/gallatinsystems/survey/dao/SurveyUtils.java

```diff
@@ -16,11 +16,13 @@
 
 package com.gallatinsystems.survey.dao;
 
+import com.google.gwt.dev.util.Pair;
```

Comment:
Why there is a need to use a `Pair` from gwt?

Reply:
we were using javafx.util.Pair but that worked in my local environment but broke the build saying the class was not found, this was another option, I don't see what si the problem with this class, can you specify? I also though about creating my own object but why doing so if that class exists, unless there is a valid reason to not use it